migrations/v2.js

```diff
@@ -0,0 +1,181 @@
+import { describe, whereContent, whereFromPlugin, mutateContent, checkContent, updatePlugin } from 'adapt-migrations';
+
+describe('adapt-contrib-assessment - v2.0.0 > v2.0.3', async () => {
+  let articles, assessments;
+
+  whereFromPlugin('adapt-contrib-assessment - from v2.0.0', { name: 'adapt-contrib-assessment', version: '<2.0.3' });
+
+  whereContent('adapt-contrib-assessment - where assessment', async content => {
+    articles = content.filter(({ _type }) => _type === 'article');
+    assessments = articles.filter(({ _type, _assessment }) => _type === 'article' && _assessment !== undefined);
+    return assessments.length;
+  });
+
+  mutateContent('adapt-contrib-assessment - add assessment._questions._canShowModelAnswer attribute', async () => {
+    assessments.forEach(assessment => {
+      assessment._questions.forEach(item => {
+        item._canShowModelAnswer = true;
+      });
+    });
+    return true;
+  });
+
+  checkContent('adapt-contrib-assessment - check assessment._questions._canShowModelAnswer attribute', async () => {
+    const isValid = assessments.every(assessment =>
+      assessment._questions.every(item => item._canShowModelAnswer)
+    );
+    if (!isValid) throw new Error('adapt-contrib-assessment - _canShowModelAnswer not added to every instance of assessment._questions');
+    return true;
+  });
+
+  updatePlugin('adapt-contrib-assessment - update to v2.0.3', { name: 'adapt-contrib-assessment', version: '2.0.3', framework: '>=2.0.0' });
+});
+
+describe('adapt-contrib-assessment - v2.0.3 > v2.1.0', async () => {
+  let articles, assessments;
+
+  whereFromPlugin('adapt-contrib-assessment - from v2.0.3', { name: 'adapt-contrib-assessment', version: '<2.1.0' });
+
+  whereContent('adapt-contrib-assessment - where assessment', async content => {
+    articles = content.filter(({ _type }) => _type === 'article');
+    assessments = articles.filter(({ _type, _assessment }) => _type === 'article' && _assessment !== undefined);
+    return assessments.length;
+  });
+
+  mutateContent('adapt-contrib-assessment - add assessment._suppressMarking', async () => {
+    assessments.forEach(assessment => {
+      assessment._suppressMarking = true;
```

Comment:
Because assessment is the article containing _assessment this is adding _suppressMarking to the article itself. 
```suggestion
      assessment._assessment._suppressMarking = true;
```